src/diffusers/models/unet_2d_condition_flax.py

```diff
@@ -214,6 +214,13 @@ def __call__(
             When returning a tuple, the first element is the sample tensor.
         """
         # 1. time
+        timesteps = timestep
```

Comment:
why ?

Reply:
This is because I copy-pasted the logic behind: https://github.com/huggingface/diffusers/blob/761f0297b08de18098e9cdbb82f36cb8911c3f8f/src/diffusers/models/unet_2d_condition.py#L216 , will change it!

Reply:
Proposed a fix in 7527ab185d744ac09948d86581bcae06e027f37c